src/components/ProfileForm/userprofileinfo.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import { Button, Row, Col } from "react-bootstrap";
+import { Auth } from "aws-amplify";
+import { API, graphqlOperation } from "aws-amplify";
+import { createUser, createDog } from "../../graphql/mutations";
+import { getUser } from "../../graphql/queries";
+
+class Userprofileinfo extends Component {
+  constructor(props) {
```

Comment:
We should try to use functional components over class components.

Reply:
Yes, at that moment, we were just trying and rushing to implement those features to work without transferring them all to functional components. However, we said about it at the end of last time. So that we will unify and also do better in coding styles in the next step's works.